src/pybind/mgr/dashboard/frontend/src/app/ceph/rgw/rgw-multisite-sync-policy-details/rgw-multisite-sync-policy-details.component.ts

```diff
@@ -0,0 +1,228 @@
+import { Component, Input, OnChanges, SimpleChanges, TemplateRef, ViewChild } from '@angular/core';
+import { NgbModalRef } from '@ng-bootstrap/ng-bootstrap';
+import { ActionLabelsI18n } from '~/app/shared/constants/app.constants';
+import { Icons } from '~/app/shared/enum/icons.enum';
+import { CdTableAction } from '~/app/shared/models/cd-table-action';
+import { CdTableColumn } from '~/app/shared/models/cd-table-column';
+import { CdTableSelection } from '~/app/shared/models/cd-table-selection';
+import { Permission } from '~/app/shared/models/permissions';
+import { ModalService } from '~/app/shared/services/modal.service';
+import { RgwMultisiteService } from '~/app/shared/api/rgw-multisite.service';
+import { CriticalConfirmationModalComponent } from '~/app/shared/components/critical-confirmation-modal/critical-confirmation-modal.component';
+import { FinishedTask } from '~/app/shared/models/finished-task';
+import { Observable, Subscriber, forkJoin as observableForkJoin } from 'rxjs';
+import { TaskWrapperService } from '~/app/shared/services/task-wrapper.service';
+import { TableComponent } from '~/app/shared/datatable/table/table.component';
+import { RgwMultisiteSyncFlowModalComponent } from '../rgw-multisite-sync-flow-modal/rgw-multisite-sync-flow-modal.component';
+import { FlowType } from '../models/rgw-multisite';
+
+@Component({
+  selector: 'cd-rgw-multisite-sync-policy-details',
+  templateUrl: './rgw-multisite-sync-policy-details.component.html',
+  styleUrls: ['./rgw-multisite-sync-policy-details.component.scss']
+})
+export class RgwMultisiteSyncPolicyDetailsComponent implements OnChanges {
+  @Input()
+  expandedRow: any;
+  @Input()
+  permission: Permission;
+
+  @ViewChild(TableComponent)
+  table: TableComponent;
+  @ViewChild('deleteTpl', { static: true })
+  deleteTpl: TemplateRef<any>;
+
+  flowType = FlowType;
+  modalRef: NgbModalRef;
+  symmetricalFlowData: any = [];
+  directionalFlowData: any = [];
+  symmetricalFlowCols: CdTableColumn[];
+  directionalFlowCols: CdTableColumn[];
+  symFlowTableActions: CdTableAction[];
+  dirFlowTableActions: CdTableAction[];
+  symFlowSelection = new CdTableSelection();
+  dirFlowSelection = new CdTableSelection();
+
+  constructor(
+    private actionLabels: ActionLabelsI18n,
+    private modalService: ModalService,
+    private rgwMultisiteService: RgwMultisiteService,
+    private taskWrapper: TaskWrapperService
+  ) {
+    this.symmetricalFlowCols = [
+      {
+        name: 'Name',
+        prop: 'id',
+        flexGrow: 1
+      },
+      {
+        name: 'Zones',
+        prop: 'zones',
+        flexGrow: 1
+      }
+    ];
+    this.directionalFlowCols = [
+      {
+        name: 'Source Zone',
+        prop: 'source_zone',
+        flexGrow: 1
+      },
+      {
+        name: 'Destination Zone',
+        prop: 'dest_zone',
+        flexGrow: 1
+      }
```

Comment:
Why are we not showing name of directional flow?
```suggestion
      {
        name: 'Name',
        prop: 'id',
        flexGrow: 1
      },
      {
        name: 'Source Zone',
        prop: 'source_zone',
        flexGrow: 1
      },
      {
        name: 'Destination Zone',
        prop: 'dest_zone',
        flexGrow: 1
      }
```

Reply:
we are not getting flow name from the API, there is already a bz opened for this issue.
That's why we are disabling the directional flow edit  delete functionality for now!

Reply:
mentioned this in my PR description already!

Reply:
Ah I missed it!